packages/shared/src/components/sidebar/Sidebar.spec.tsx

```diff
@@ -104,7 +104,7 @@ const renderComponent = (
   );
 };
 
-it('should remove alert dot for filter alert when there is a pre-configured feedSettings', async () => {
+it('should remove feed filter button once user has filters', async () => {
```

Comment:
I think this test was reused for something else. This is for making the alerts go away, the right test should be just checking if the button is gone on load if the user has filters.

Reply:
This test is to specifically see if the mutation is still called.
I've added another test to check if the button changes it's this one:
`should show the my feed items if the user has filters`.

Let me know if that makes sense?

Reply:
This mutation should be executed once the user has clicked the close button if I'm not mistaken. It would be the `alert.filter` to decide whether a message is displayed. The feed filter button is decided by `userFeedSettings`.